core/jazz_ui/src/app/secondary-components/create-service/oss/create-service.component.ts

```diff
@@ -198,6 +222,12 @@ export class CreateServiceComponent implements OnInit {
     this.dynamoStreamString = "arn:aws:dynamo:" + this.regionSelected + ":" + this.accountSelected + ":table/";
   }
 
+  azureEventsPrefix() {
+    this.documentDBStreamString = "subscriptions/fd312e60-798b-4933-a4c9-66fa2697a464/providers/Microsoft.cosmosdb/"
```

Comment:
Why is this subscriptionId hardcoded? Please use configuration for this kind of information.

Reply:
Changed hardcoded values and fetching from config